core-primitives/stf-state-handler/src/test/sgx_tests.rs

```diff
@@ -373,6 +351,18 @@ fn given_hello_world_state() -> StfState {
 	state
 }
 
+fn setup(
```

Comment:
How about giving it a bit more descriptive name?
```suggestion
fn setup_test_directory(
```

Reply:
Agreed ,it is not that descriptive, however it also creates and initializes the state key repository, so I think it should state generic IMO. As a compromise I went for `test_setup`, I hope you can live with that. :)